src/components/Button.js

```diff
@@ -2,21 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Calculator.css';
 
-const Button = ({ label }) => {
```

Comment:
![image](https://github.com/CVILLA09/Math-Magicians/assets/93039359/8dca2106-c4dd-4f6d-8902-1f9dacb1d6ff)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you add the correct title of your project instead of `About the project` into the readme file to make your readme descriptive and professional 👍